lms/static/sass/base/_headings.scss

```diff
@@ -88,29 +88,33 @@ $headings-base-color:                       $gray-d2;
     letter-spacing: 1px;
 }
 
-.xblock .xblock {
+// let's make these classnames available to the entire LMS!
+
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6,
+.hd-7,
+.hd-8 {
+    @extend %reset-headings;
+}
+
 
-    .hd-1,
-    .hd-2,
-    .hd-3,
-    .hd-4,
-    .hd-5,
-    .hd-6,
-    .hd-7,
-    .hd-8 {
-        @extend %reset-headings;
+// ----------------------------
+// #CANNED
+// ----------------------------
+// canned heading classes
+@for $i from 1 through $headings-count {
+    .hd-#{$i} {
+        @extend %hd-#{$i};
     }
+}
 
+// H3 was problematic in xblocks, we so we'll keep it as it was
 
-    // ----------------------------
-    // #CANNED
-    // ----------------------------
-    // canned heading classes
-    @for $i from 1 through $headings-count {
-        .hd-#{$i} {
-            @extend %hd-#{$i};
-        }
-    }
+.xblock .xblock {
```

Comment:
These H3s are funky and some of them need to be scoped to xblocks, we I'm keeping this particular H3 scoped.